components/core/src/streaming_archive/writer/Archive.hpp

```diff
@@ -23,6 +24,8 @@
 #include "../ArchiveMetadata.hpp"
 #include "../MetadataDB.hpp"
 
+
+
```

Comment:
Can we remove the empty lines (usually you can run clang-format to automate these style checks)?